maquetado/setup/index.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML & CSS Training</title>
+    <meta name="description" content="training">
+    <meta name="author" content="Wolox">
+    <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet">
+    <link rel="stylesheet"  type="text/css" href="application.css">    
+  </head>
+  <body>
+    <div class="title">
```

Comment:
Remove this divs, you dont need this container, all styles can be in the `<h2` below